pkg/kyverno/apply/apply_command.go

```diff
@@ -141,7 +139,7 @@ func Command() *cobra.Command {
 	cmd.Flags().BoolVarP(&policyReport, "policy-report", "", false, "Generates policy report when passed (default policyviolation r")
 	cmd.Flags().StringVarP(&namespace, "namespace", "n", "", "Optional Policy parameter passed with cluster flag")
 	cmd.Flags().BoolVarP(&stdin, "stdin", "i", false, "Optional mutate policy parameter to pipe directly through to kubectl")
-	cmd.Flags().BoolVarP(&toggle.AutogenInternals, "autogenInternals", "", toggle.DefaultAutogenInternals, "Use autogen internals")
+	// cmd.Flags().BoolVarP(&toggle.AutogenInternals, "autogenInternals", "", toggle.DefaultAutogenInternals, "Use autogen internals")
```

Comment:
Do we need this for CLI?

Reply:
I couldn't find a way to easily add it to cobra, the env var should work though.

Reply:
Do you want to create a separate issue for CLI support?

Reply:
I'll create one, in any case we can configure the feature flag using an env var. I will look at cobra to see how to support it.

Reply:
Sounds good, thanks!

Reply:
I created https://github.com/kyverno/kyverno/issues/3464